src/modules/i18n/locales/en.json

```diff
@@ -484,6 +484,10 @@
     "time_hide": "Disable representation of data time stamps.",
     "time_select_year": "Select a year",
     "time_show": "Enable representation of data time stamps.",
+    "time_slider_legend_tippy_intro": "The time slider displays different colours depending on whether the time layers have data for the selected year or not.",
```

Comment:
In the UI we never talk about `layer` but about `map` so we should keep this nomenclature in help text.

```
The time slider displays different colors depending on whether the maps have data for the year or not.
```

For German and Italian, @hansmannj and @Luke252 can maybe help and also check english 😉 . For french a proposal that can be double check by you and @pakb 

```
Le curseur de temps affiche différentes couleurs selon si les cartes temporelles disposent de données ou non.
```

Reply:
@ltshb I checked and compared the IT and EN versions with the rest. Hope it's clear ;)